src/components/vm/consoles/serialConsole.jsx

```diff
@@ -89,13 +89,13 @@ class SerialConsoleCockpit extends React.Component {
 
         return (
             <>
-                <div className="pf-c-console__actions-serial">
+                <div className="pf-v5-c-console__actions-serial">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1162-20230808-102821-f5d4e2e6-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)